cli/command/container/wait.go

```diff
@@ -28,6 +29,10 @@ func NewWaitCommand(dockerCli *command.DockerCli) *cobra.Command {
 			return runWait(dockerCli, &opts)
 		},
 	}
+
+	flags := cmd.Flags()
+	flags.StringVar(&opts.condition, "condition", "not-running", "Wait until a container state reaches the given condition, either 'not-running' (default), 'next-exit', or 'removed'")
```

Comment:
I remember doing something in the past where I wanted to wait for a container on the CLI that was only "created" but I can't seem to remember why.

If we think it's just clutter then I can remove it, it would also remove the man-page changes and  some integration-cli tests that I added.